shared/signup/phone-number/phone-input.tsx

```diff
@@ -218,8 +218,10 @@ type Props = {
   autoFocus?: boolean
   defaultCountry?: string
   onChangeNumber: (phoneNumber: string) => void
-  onChangeValidity: (valid: boolean) => void
+  onChangeValidity: (valid: boolean, phoneNumber: string) => void
   onEnterKeyDown?: () => void
+  // A ReactNode to display beneath and aligned to the number input box
+  result?: React.ReactNode
```

Comment:
Hm, I don't think it makes sense for `PhoneInput` to add this. Can `PhoneSearch` render the result below directly?

Reply:
The issue is that the [design](https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/5d2775a1dadd1f77b1768116) calls for having the person aligned with the number input box rather than directly underneath it. I confirmed with Cecile that she does want it aligned in that way and when I asked in #frontend about the best way to implement it, this is what Song suggested. How would you implement it?

Reply:
I guess I'd do it this way. @cecileboucheron do you think we could align it on mobile the same way as on desktop? The user result area moves as a result of changing the country code as it is, which seems weird.

Reply:
Additionally, @cecileboucheron the desktop design didn't show where the person should be aligned? I decided it seemed to make more sense to have the person left aligned on the desktop since it seemed a little squished otherwise but curious to get your opinion.

Reply:
Desktop for reference:
<img width="464" src="https://user-images.githubusercontent.com/11968340/63442824-c162cd80-c401-11e9-9da1-3767084ddd7a.png">

Reply:
Sure, all left-aligned sounds good to me.